plugins/tiddlywiki/browser-sniff/browser.js

```diff
@@ -42,8 +42,12 @@ exports.getInfoTiddlerFields = function() {
 				["is/sailfish","sailfish"],
 				["is/android","android"],
 				["is/windowsphone","windowsphone"],
-				["is/firefoxos","firefoxos"]
+				["is/firefoxos","firefoxos"],
+				["is/mobile","mobile"]
 			];
+		$tw.browser = $tw.utils.extend($tw.browser, {
+			mobile: bowser.browser.mobile,
```

Comment:
@Jermolene Other people can pr if they need other variables

Reply:
Thank you @linonetwo.

I notice that we give special treatment to `bowser.browser.mobile` by exposing it through `$tw.browser.mobile`. But perhaps it makes sense to expose all of the bowser properties that we've selected, perhaps as `$tw.browser.is[xxxx]` . For example `$tw.browser.is.firefoxos, and deprecate the existing `$tw.browser.isIE` and `$tw.browser.isFirefox`.